config/locales/en.yml

```diff
@@ -32,6 +32,7 @@ en:
     support: Support Us
     search: Search
     about: About
+    locale_articles: "English's articles"
```

Comment:
We'll also need this exact same key in every YAML locale file. Even if we don't have a translation yet. Otherwise we get a "translation missing" error.